### PR TITLE
Suscribe, Terms and Privacy Policies CheckBox added.

### LINKED ISSUE #10: Suscribe, Terms and Privacy Policies checkbox
# REQUIREMENTS
It must be conformed by two checkbox

# DESIGN
image

# Acceptance Criteria
- [ ] Be available to select and unselect the checkbox

### PR DESCRIPTION
**Pull request by yovanha**
# Description
What did you implemented/Why did you implemented this:
Create two checkbox components where the user can select and unselect them.
 
 Examples:
 - What?: I've created a checkbox component for Suscribe and Terms and Privacy Policy, For more background, see ticket #10  Why?: These changes complete the user login and account creation experience. See #10 
 
  ## Screenshots 
image
image

### CODE REVIEW

File: source/entities/signUp/atoms/CheckBoxElements.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                />
+                <Text style={styles.checkBoxText}>I agree to the <Text style = {{textDecorationLine: 'underline'}}>Terms</Text> and <Text style = {{textDecorationLine: 'underline'}}>Privacy Policy.</Text></Text>
+                <Text style={styles.alert}>*</Text>
+            </View>
+
+            <View style={styles.checkboxContainer}>
+                <CheckBox
+                checked={suscribeSelected}
+                onPress={() => suscribeSelection(!suscribeSelected)}
+                />
```
Comment (McAdam17) [inline source/entities/signUp/atoms/CheckBoxElements.js:23]: You didn't add a tabulation here, but you did above

**Review by McAdam17 — COMMENTED** [review]
The rest is fine in my opinion

File: source/entities/signUp/atoms/CheckBoxElements.js (hunk 1 above)
Comment (yovanha) [reply source/entities/signUp/atoms/CheckBoxElements.js:23]: Thanks for the observation, I will make the change.

**Review by angel-sm — CHANGES_REQUESTED** [review]
Great! just need to change this component to Molecules folder

**Review by McAdam17 — APPROVED** [review]
well done

**Review by angel-sm — APPROVED** [review]

**Pull request closed without merging**